### DISCUSSION TITLE
One line tables in status total headers

### DISCUSSION THREAD

**Discussion by ghost** [sni/thruk#1321] [Show and tell]
Hi, default status header (I known there is a button to hide top row) is too big for me (in computer web browser).
I changed templates to make one line table (with custom css extracted from tabler.css).
And changed the conditions to make if some metric is null, then hide it.
Attached tt files and screenshot.

Tell me if you like it and if you are using it.

![single-line-global-status](https://github.com/sni/Thruk/assets/61549534/e92ac4a5-9b0d-424d-bd58-3f9da3cbee97)
[_host_status_details_box.tt.txt](https://github.com/sni/Thruk/files/13510060/_host_status_details_box.tt.txt)
[_service_status_details_box.tt.txt](https://github.com/sni/Thruk/files/13510061/_service_status_details_box.tt.txt)

**Comment by sni** [comment]
i like the idea
